fronts-client/src/actions/Cards.ts

```diff
@@ -206,12 +206,17 @@ const updateCardMetaWithPersist = (persistTo: PersistTo) =>
 		persistTo,
 	});
 
-const minimumGroupBoostLevel = (groupId: string) => {
-	// boost is the smallest boost level for `Big`
-	if (groupId === '1') return 'boost';
-	// megaboost is the smallest boost level for `Very Big`
-	if (groupId === '2') return 'megaboost';
-	return 'default';
+const minimumGroupBoostLevel = (groupName: string) => {
+	switch (groupName) {
+		case 'very big':
+			return 'megaboost';
+		case 'big':
+			return 'boost';
+		case 'splash':
+		case 'standard':
+		default:
+			return 'default';
+	}
```

Comment:
This refactor follows the same pattern as https://github.com/guardian/facia-tool/blob/ffd6ea58094daedbddaa09ce064f163ab276c3d8/fronts-client/src/util/card.ts#L343